pkg/controllers/management/restrictedadminrbac/register.go

```diff
@@ -32,9 +40,42 @@ func Register(ctx context.Context, management *config.ManagementContext, wrangle
 		roleBindings:       management.RBAC.RoleBindings(""),
 		rbLister:           management.RBAC.RoleBindings("").Controller().Lister(),
 		fleetworkspaces:    management.Management.FleetWorkspaces(""),
+		clusters:           management.Wrangler.Mgmt.Cluster(),
+		crtbCache:          management.Wrangler.Mgmt.ClusterRoleTemplateBinding().Cache(),
+		crtbCtrl:           management.Wrangler.Mgmt.ClusterRoleTemplateBinding(),
+		grbCache:           management.Wrangler.Mgmt.GlobalRoleBinding().Cache(),
 	}
-
+	r.globalRoleBindings.AddHandler(ctx, grbHandlerName, r.clusterOwnerSync)
 	r.globalRoleBindings.AddHandler(ctx, "restrictedAdminGlobalBindingsFleet", r.ensureRestricedAdminForFleet)
+
+	// if a fleetwokspace changes then e
```

Comment:
Nit: I think this was meant to end with "enqueue".

Reply:
Fixed.